### PR TITLE
Change some System.Net.Requests tests to use LoopbackServer

### PR DESCRIPTION
**Pull request by davidsh**
Changed HttpWebResponse test to use LoopbackServer.

More tests will be changed to use this server as part of
increasing test reliability and simplifying the requirements for the Azure test server.

### CODE REVIEW

**Comment by davidsh** [comment]
cc: @stephentoub @CIPop

File: src/System.Net.Requests/tests/HttpWebResponseTest.cs [hunk 1]
```diff
@@ -24,10 +24,25 @@ public class HttpWebResponseTest
...
-            HttpWebRequest request = WebRequest.CreateHttp(HttpTestServers.RemoteEmptyContentServer);
-            request.Method = HttpMethod.Get.Method;
-            WebResponse response = await request.GetResponseAsync();
-            Assert.Equal(string.Empty, response.ContentType);
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                HttpWebRequest request = WebRequest.CreateHttp(url);
+                request.Method = HttpMethod.Get.Method;
+                Task<WebResponse> getResponse = request.GetResponseAsync();
+                {
```
Comment (stephentoub) [inline src/System.Net.Requests/tests/HttpWebResponseTest.cs]: Nit: unnecessary braces/indentation (but if this was on purpose, it's fine)

File: src/System.Net.Requests/tests/project.json [hunk 2]
```diff
@@ -4,8 +4,11 @@
     "System.IO.Compression": "4.1.0-rc3-24113-00",
     "System.Linq.Expressions": "4.1.0-rc3-24113-00",
     "System.Net.Http": "4.1.0-rc3-24113-00",
+    "System.Net.NetworkInformation": "4.1.0-rc3-24113-00",
     "System.Net.Primitives": "4.0.11-rc3-24113-00",
     "System.Net.Requests": "4.0.11-rc3-24113-00",
+    "System.Net.Security": "4.0.0-rc3-24113-00",
+    "System.Net.Sockets": "4.1.0-rc3-24113-00",
```
Comment (stephentoub) [inline src/System.Net.Requests/tests/project.json]: By the time you see this, I will have hopefully merged the package update to fix the SslStream issues.  At that point you'll need to manually bump these new entries up from 113 to 116.

**Comment by stephentoub** [comment]
LGTM

File: src/System.Net.Requests/tests/HttpWebResponseTest.cs (hunk 1 above)
Comment (davidsh) [inline src/System.Net.Requests/tests/HttpWebResponseTest.cs]: Removed. It was cut/paste error.

**Comment by stephentoub** [comment]
Test Innerloop CentOS7.1 Debug Build and Test please
Test Innerloop Windows_NT Debug Build and Test please

**Pull request merged**